migrations/sql-annex/V2018.02.15.1500__RecreateBurdenEstimateStochasticWithSmallIntFields.sql

```diff
@@ -0,0 +1,24 @@
+DROP TABLE burden_estimate_stochastic;
+
+CREATE TABLE burden_estimate_stochastic (
+  id BIGSERIAL NOT NULL,
+  burden_estimate_set integer NOT NULL,
+  country text NOT NULL,
```

Comment:
Once we change the API to populate country numeric id, we can remove this column and change the constraint below